### DISCUSSION TITLE
How could I test my server api by tools like postman?

### DISCUSSION THREAD

**Discussion by s3xysteak** [nuxt/nuxt#29750] [Questions]
I tried `localhost:3000/api/test` but it seems routing to app routes 😢.

**Comment by weloin-sukanta** [comment]
@s3xysteak, not exactly sure about what you are facing, but many times, I've experienced that when there is an error in the API, it gives nuxt's front-end error response.

A few times I've noticed that, there is no error yet showing error response, then restarting and/or cleaning solves such problems.

**Comment by xjccc** [comment]
I test api with apifox like this. Its works for me
![image](https://github.com/user-attachments/assets/4028b98e-2121-4f22-8dd0-f0888a16c213)

**Comment by abid365** [comment]
<img width="1917" height="1025" alt="image" src="https://github.com/user-attachments/assets/53bddba3-0777-4623-a833-6efdb7d5b99f" />

I'm using httpie, it works perfectly. Just point out your api end point with the localhost:<your-port>
